File: server/src/main/java/WAT/I8E2S4/TaskManager/Model/Task.java
package WAT.I8E2S4.TaskManager.Model;

import WAT.I8E2S4.TaskManager.Model.Category;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.*;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
@Table(name="task")
public class Task {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Getter @Setter private long id;

    @Getter @Setter private String name;
    @Getter @Setter private String description;
    @JsonFormat(pattern = "dd/MM/yyyy HH:mm")
    @Getter @Setter private LocalDateTime startDateTime;
    @JsonFormat(pattern = "dd/MM/yyyy HH:mm")
    @Getter @Setter private LocalDateTime endDateTime;
    @Getter @Setter private boolean active;
    @Getter @Setter private boolean notification;

    @ManyToOne()
    @Getter @Setter Category category;
}

File: mobileClient/app/src/main/java/I8E2S4/mobileTaskManager/model/Task.kt
package I8E2S4.mobileTaskManager.model

class Task{
    var id: Long? = null
    var name: String? = null
    var description: String? = null
    var startDateTime: String? = null
    var endDateTime: String? = null
    var active: Boolean? = null
    var notification: Boolean? = null
    override fun toString(): String{
        return name?:""
    }
}